style/style.css

```diff
@@ -0,0 +1,14 @@
+.wmf-wp-with-preview {
```

Comment:
This file should probably be renamed. I don't have a good suggestion at the moment but style.css doesn't say anything about what this is. Same with css.js.